x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/maintenance_window_callout/maintenance_window_callout.test.tsx

```diff
@@ -0,0 +1,223 @@
+    { gte: '2024-03-22T11:36:14.028Z', lte: '2024-03-22T11:37:00.000Z' },
+    { gte: '2024-03-29T11:36:14.028Z', lte: '2024-03-29T11:37:00.000Z' },
+    { gte: '2024-04-05T10:36:14.028Z', lte: '2024-04-05T10:37:00.000Z' },
+    { gte: '2024-04-12T10:36:14.028Z', lte: '2024-04-12T10:37:00.000Z' },
+    { gte: '2024-04-19T10:36:14.028Z', lte: '2024-04-19T10:37:00.000Z' },
+  ],
+  id: '5eafe070-e030-11ed-ac81-bfcb4982b4fd',
+  status: 'upcoming',
+  expiration_date: '2024-04-21T10:36:26.999Z',
+  r_rule: {
+    dtstart: '2023-04-21T10:36:14.028Z',
+    tzid: 'Europe/Amsterdam',
+    freq: 3,
+    interval: 1,
+    byweekday: ['FR'],
+  },
+  created_by: 'elastic',
+  updated_by: 'elastic',
+  created_at: '2023-04-21T10:36:26.999Z',
+  updated_at: '2023-04-21T10:36:26.999Z',
+  event_start_time: '2023-04-28T10:36:14.028Z',
+  event_end_time: '2023-04-28T10:37:00.000Z',
+};
+
+describe('MaintenanceWindowCallout', () => {
+  let appToastsMock: jest.Mocked<ReturnType<typeof useAppToastsMock.create>>;
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+
+    appToastsMock = useAppToastsMock.create();
+    (useAppToasts as jest.Mock).mockReturnValue(appToastsMock);
+  });
+
+  afterEach(() => {
+    cleanup();
+    jest.restoreAllMocks();
+  });
+
+  it('MW callout should be visible if currently there is at least one "running" maintenance window', async () => {
+    (fetchActiveMaintenanceWindows as jest.Mock).mockResolvedValue([RUNNING_MAINTENANCE_WINDOW_1]);
+
+    const { findByText } = render(<MaintenanceWindowCallout />, { wrapper: TestProviders });
+
+    expect(await findByText('A maintenance window is currently running')).toBeInTheDocument();
+  });
+
+  it('A single MW callout should be visible if currently there are multiple "running" maintenance windows', async () => {
+    (fetchActiveMaintenanceWindows as jest.Mock).mockResolvedValue([
+      RUNNING_MAINTENANCE_WINDOW_1,
+      RUNNING_MAINTENANCE_WINDOW_2,
+    ]);
+
+    const { findAllByText } = render(<MaintenanceWindowCallout />, { wrapper: TestProviders });
+
+    expect(await findAllByText('A maintenance window is currently running')).toHaveLength(1);
+  });
+
+  it('MW callout should NOT be visible if currently there are no active (running or upcoming) maintenance windows', async () => {
+    (fetchActiveMaintenanceWindows as jest.Mock).mockResolvedValue([]);
+
+    const { container } = render(<MaintenanceWindowCallout />, { wrapper: TestProviders });
+
+    expect(container).toBeEmptyDOMElement();
+  });
+
+  it('MW callout should NOT be visible if currently there are no "running" maintenance windows', async () => {
+    (fetchActiveMaintenanceWindows as jest.Mock).mockResolvedValue([UPCOMING_MAINTENANCE_WINDOW]);
+
+    const { container } = render(<MaintenanceWindowCallout />, { wrapper: TestProviders });
+
+    expect(container).toBeEmptyDOMElement();
+  });
+
+  it('User should see an error toast if there was an error while fetching maintenance windows', async () => {
```

Comment:
nit: to be consistent with the test names above `should show an error toast if there was an error while fetching maintenance windows`.

Reply:
Thanks, pushed